samples/betaFeatures.js

```diff
@@ -0,0 +1,586 @@
+  const speech = require('@google-cloud/speech').v1p1beta1;
+
+  // Creates a client
+  const client = new speech.SpeechClient();
+
+  const config = {
+    encoding: `FLAC`,
+    sampleRateHertz: 16000,
+    languageCode: `en-US`,
+    enableWordTimeOffsets: true,
+    enableWordConfidence: true,
+  };
+
+  const audio = {
+    uri: gcsUri,
+  };
+
+  const request = {
+    config: config,
+    audio: audio,
+  };
+
+  client
+    .recognize(request)
+    .then(data => {
+      const response = data[0];
+      const transcription = response.results
+        .map(result => result.alternatives[0].transcript)
+        .join('\n');
+      const confidence = response.results
+        .map(result => result.alternatives[0].confidence)
+        .join(`\n`);
+      console.log(
+        ` Transcription: ${transcription} \n Confidence: ${confidence}`
+      );
+
+      console.log(` Word-Level-Confidence:`);
+      const words = response.results.map(result => result.alternatives[0]);
+      console.log(`WORDS: `);
+      words.forEach(a => console.log(JSON.stringify(a, null, 4)));
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END speech_transcribe_word_level_confidence_gcs]
+}
+
+//audio file paths for Defaults
+const path = require(`path`);
+const monoFileName = `commercial_mono.wav`;
+const monoFilePath = path.join(
+  __dirname,
+  `../samples/resources/${monoFileName}`
+);
+// const defaultURI = 'gs://cloud-samples-tests/speech/commercial_mono.wav';
+const stereoFileName = `commercial_stereo.wav`;
+const stereoFilePath = path.join(
+  __dirname,
+  `../samples/resources/${stereoFileName}`
+);
+const multiLanguageFileName = `multi.wav`;
+const multiLanguageFile = path.join(
+  __dirname,
+  `../samples/resources/${multiLanguageFileName}`
+);
+//const multiLangURI = `gs://cloud-samples-tests/speech/multi.wav`;
+
+// const defaultStereoURI =
+//   'gs://cloud-samples-tests/speech/commercial_stereo.wav';
+const gnome = `/Users/cfrater/SPEECH3/nodejs-speech/samples/resources/Google_Gnome.wav`;
+
+const Brooklyn = 'brooklyn.flac';
+const BrooklynFilePath = path.join(
+  __dirname,
+  `../samples/resources/${Brooklyn}`
+);
+
+require(`yargs`)
+  .demand(1)
+  .command(
+    `Diarization`,
+    `Isolate distinct speakers in an audio file`,
+    {},
+    opts => speechTranscribeDiarization(opts.speechFile)
+  )
+  .command(
+    `DiarizationGCS`,
+    `Isolate distinct speakers in an audio file located in a Google Cloud Storage bucket.`,
+    {},
+    opts => asyncSpeechTranscribeDiarizationGCS(opts.gcsUri)
+  )
+  .command(
+    `multiChannelTranscribe`,
+    `Differentiates input by audio channel in local audio file.`,
+    {},
+    opts => speechTranscribeMultiChannel(opts.speechFileGnome)
+  )
+  .command(
+    `multiChannelTranscribeGCS`,
+    `Differentiates input by audio channel from GCS audio file.`,
+    {},
+    opts => speechTranscribeMultichannelGCS(opts.gcsUriStereo)
```

Comment:
Running this, I only see `Channel Tag 0`...

Reply:
noted in conversation 7/23; reuploading audio file as java sample works